tensorflow_addons/optimizers/adabelief.py

```diff
@@ -0,0 +1,353 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""AdaBelief optimizer."""
+
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow as tf
+from tensorflow_addons.utils.types import FloatTensorLike
+
+from typing import Union, Callable, Dict
+
+
+@tf.keras.utils.register_keras_serializable(package="Addons")
+class AdaBelief(tf.keras.optimizers.Optimizer):
+    """Variant of the Adam optimizer that achieves fast convergence,
```

Comment:
Single line description in the first line e.g. `Variant of the Adam Optimizer.` Then a blank line and then the long description.